src/utils.js

```diff
@@ -1,5 +1,26 @@
 import { backupProviderAPIQueryStrings } from './popup/helper';
 
+export function showModal(modalText, onModalConfirm, onModalClose, elements) {
+  const {
+    modal,
```

Comment:
There is no need to pass `elements` as the argument. Instead, import and use it.